packages/amplify-category-api/provider-utils/awscloudformation/appsync-schemas/many-relationship-schema.graphql

```diff
@@ -1,16 +1,18 @@
 type Blog @model {
   id: ID!
   name: String!
-  posts: [Post] @connection(name: "BlogPosts")
+  posts: [Post] @connection(keyName: "byBlog", fields: ["id"])
 }
-type Post @model {
+
+type Post @model @key(name: "byBlog", fields: ["blogID"]) {
   id: ID!
   title: String!
-  blog: Blog @connection(name: "BlogPosts")
-  comments: [Comment] @connection(name: "PostComments")
+  blogID: String!
+  comments: [Comment] @connection(keyName: "byPost", fields: ["id"])
 }
-type Comment @model {
+
+type Comment @model @key(name: "byPost", fields: ["postID", "content"]) {
   id: ID!
-  content: String
-  post: Post @connection(name: "PostComments")
-}
\ No newline at end of file
+  postID: ID!
+  content: String!
+}
```

Comment:
We should also add the "belongs to" connection into the Comment model definition just to have the same behavior as the previous setup.
`post: Post @connection(fields: ["postID"])`

Reply:
Changed